articles/multifactor-authentication/index2.md

```diff
@@ -15,49 +15,45 @@ Multifactor Authentication (MFA) is a method of verifying a user's identity by r
 
 Auth0 supports the following methods of implementing MFA:
```

Comment:
Before we replace all the current docs (that do not mention Guardian as it is not officially announced) the old MFA docs live on index, when we officially switch to the new docs on index2 will become the new index.